plugins/plugin-hosted/codenvy-hosted-ext-factory/src/main/java/com/codenvy/ide/factory/client/utils/FactoryProjectImporter.java

```diff
@@ -132,6 +132,10 @@ public void apply(List<ProjectConfigDto> projectConfigs) throws OperationExcepti
                 Set<String> projectNames = new HashSet<>();
                 String createPolicy = factory.getPolicies() != null ? factory.getPolicies().getCreate() : null;
                 for (ProjectConfigDto projectConfig : projectConfigs) {
+                    if (projectConfig.getSource().getLocation() == null) {
+                        continue;
```

Comment:
I understood that it is possible only if project is subproject and it is not need to import it. Please add this comment here